ui/DesignSystem/Component/ProjectCard.svelte

```diff
@@ -1,11 +1,15 @@
-<script>
+<script lang="ts">
   import { Icon } from "../Primitive";
   import Badge from "../Component/Badge.svelte";
+  import { BadgeType } from "../../src/badge";
 
-  export let title = null;
-  export let description = null;
-  export let showRegisteredBadge = false;
-  export let badge = null;
+  export let title: string;
+
+  export let description = "";
+
+  /* eslint-disable @typescript-eslint/no-inferrable-types */
```

Comment:
This is annoying, but `svelte-check` needs the explicit type annotation for `boolean` props with default values.

Reply:
We should move your comment to an inline comment so other collaborators know why we have to leave it in.